spec/models/registration_spec.rb

```diff
@@ -3,6 +3,8 @@
 require 'spec_helper'
 
 describe 'Registration' do
+  let(:conference) { create(:conference) }
+  let(:registration1) { create(:registration, conference: conference, user: user) }
```

Comment:
You also need to declare a user instance ;) (See your failing tests ...)
And also add an empty line between the let and the describe!